src/legacy/core_plugins/kibana/public/kibana.js

```diff
@@ -38,17 +38,22 @@ import 'uiExports/shareContextMenuExtensions';
 import 'uiExports/interpreter';
 
 import 'ui/autoload/all';
-import './management';
+
 import { localApplicationService } from './local_application_service';
 
 npSetup.plugins.kibanaLegacy.registerLegacyAppAlias('doc', 'discover', { keepPrefix: true });
 npSetup.plugins.kibanaLegacy.registerLegacyAppAlias('context', 'discover', { keepPrefix: true });
 
+npSetup.plugins.kibanaLegacy.forwardApp('management', 'management', (path) => {
+  return path.replace('/management', '');
+});
+
```

Comment:
Can we move this legacy app to the kibana_legacy plugin in KP? That way the redirects don't require a second full-page refresh. Could be done in a follow up.

\cc @flash1293